Course_Part_06/src/Assignment22ClassEmployee.java

```diff
@@ -0,0 +1,20 @@
+public class Assignment22ClassEmployee {
+
+    public static void main(String[] args) {
+
+        Employee pieter = new Employee(1,"Pieter", "Post", 2500);
+        pieter.printAnnualSalary();
```

Comment:
Hier is de bedoeling om een getter te gebruiken om het salaris op te halen. Vervolgens een println te gebruiken om het salaris te printen